api/room/schema.py

```diff
@@ -94,11 +110,10 @@ def resolve_rooms(self, info, **kwargs):
         query = Room.get_query(info)
         exact_query = room_filter(query, filter_data)
         if not page:
-            return exact_query.order_by(func.lower(RoomModel.name)).all()
+            return exact_query.all()
         page = validate_page(page)
         self.query_total = exact_query.count()
-        result = exact_query.order_by(func.lower(
-            RoomModel.name)).limit(per_page).offset(page*per_page)
+        result = exact_query.limit(per_page).offset(page*per_page)
```

Comment:
Same applies here. This was meant to order the names returned after pagination. See the query below
`query {
  allRooms(page:1, perPage:1){
   rooms{
      name
   }
   hasNext
   hasPrevious
   pages
}
}`.

Reply:
Same applies here